homeassistant/components/airzone/__init__.py

```diff
@@ -0,0 +1,65 @@
+"""The Airzone integration."""
+from __future__ import annotations
+
+from aioairzone.common import ConnectionOptions
+from aioairzone.localapi_device import AirzoneLocalApi
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PORT, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import aiohttp_client
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, MANUFACTURER
+from .coordinator import AirzoneUpdateCoordinator
+
+PLATFORMS: list[Platform] = [Platform.SENSOR]
+
+
+class AirzoneEntity(CoordinatorEntity):
+    """Define an Airzone entity."""
+
+    def __init__(
+        self,
+        coordinator: AirzoneUpdateCoordinator,
+        entry: ConfigEntry,
+        system_zone_id: str,
+        zone_name: str,
+    ) -> None:
+        """Initialize."""
+        super().__init__(coordinator)
+
+        self._attr_device_info: DeviceInfo = {
+            "identifiers": {(DOMAIN, f"{entry.entry_id, system_zone_id}")},
```

Comment:
{entry.entry_id}_{system_zone_id}

Reply:
oops :$

Reply:
Fixed in 75388356e7e41fedc3c5d89477a6045cffbac38b
Sorry!